helm/templates/rbac.yaml

```
@@ -169,6 +169,7 @@ apiVersion: rbac.authorization.k8s.io/v1
 kind: RoleBinding
 metadata:
   name: operand-deployment-lifecycle-manager
+  namespace: {{ .Values.operatorNamespace }}
```

```suggestion
  namespace: {{ .Values.global.operatorNamespace }}
```

Actually, the latest code from master branch has already fixed this

---

Yes, I will do the rebase.